web/src/pages/transaction/add.vue

```diff
@@ -3,12 +3,14 @@
 const store = ref("")
 const amount = ref(0)
 const date = ref(Date.now())
+const transactionType = ref('expense');
```

Comment:
Since there are only ever two transaction types, would it make more sense to be a boolean? Something like `isExpense`? It can still be represented as two options in the selection menu like it is now, but with `true/false` as the `value`s.

Reply:
From testing and experimenting, I think it makes more sense to keep the code as it is, there may be more interesting transactions in the future.